### DISCUSSION TITLE
I can't install atom on windows 7, it prompt issues on windows update?

### DISCUSSION THREAD

**Discussion by justinhky** [atom/atom#22253] [Q&A]
Hello,

I come across this issue when I trying to install atom on my home laptop. It prompts issues with windows update. My laptop is windows 7 32 bit. If there is a solution, please let me know and close this topic. Thanks in advance.
![prompt20210421233008](https://user-images.githubusercontent.com/58619443/115580805-025b6580-a2fa-11eb-90fb-1d8e713370ea.png)

**Comment by sadick254** [comment]
Which version of Atom were you installing? Could you try installing [version 1.55](https://github.com/atom/atom/releases/tag/v1.55.0) and see if you get the same prompt.
